### PR TITLE
Update FAQ cert_recycle_bin: Mention new display of permanent deletion

### PR DESCRIPTION
**Pull request by larswmh**
EN:
image

DE:
image

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -329,6 +341,7 @@
                         "textblock": [
                             "Starting with version 2.13, <b>vaccination, recovery, or test certificates</b> that you have deleted are temporarily moved to a recycle bin.  This allows you to easily recover the certificate in case of accidental removal and move it back to its original location. If a certificate remains in the recycle bin for 30 days, it will automatically be permanently deleted, unless you have manually permanently deleted it from the recycle bin beforehand. A certificate that has been deleted in the app can still be rescanned. If it is still in the recycle bin and has therefore not been permanently removed, you will receive a message that it has been restored.",
                             "Starting with version 2.14, <b>tests</b> that you have deleted will also be moved to the recycle bin for 30 days.",
+                            "Starting with version 2.17, the time at which an item in the recycle bin is finally deleted is displayed below the respective item.",
```
Comment (MikeMcC399) [inline src/data/faq.json:344]: "is finally deleted" =>

"will be finally deleted"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Grammar suggestion

File: src/data/faq.json (hunk 1 above)
Comment (larswmh) [reply src/data/faq.json]: Fixed in the latest commit, thanks!

**Comment by MikeMcC399** [comment]
@larswmh 

This PR seems to have other changes in it. Is the `release/2.17` branch rebased onto the `master `branch and this branch rebased on to the `release/2.17` branch?

**Comment by larswmh** [comment]
> @larswmh
> 
> This PR seems to have other changes in it. Is the `release/2.17` branch rebased onto the `master `branch and this branch rebased on to the `release/2.17` branch?

I've accidently based this branch on master before and targeted the 2.17 branch which did not have these changes at the time. The rebase seems to have fixed this now

**Review by dsarkar — APPROVED** [review]

**Pull request merged**